K2Bridge/KustoDAL/KustoDataAccess.cs

```diff
@@ -93,7 +94,10 @@ public async Task<IndexListResponseElement> GetIndexListAsync(string indexName)
                 using IDataReader kustoTables = await Kusto.ExecuteControlCommandAsync(readTablesCommand, RequestContext);
                 if (kustoTables != null)
                 {
-                    MapIndexList(kustoTables, response);
+                    while (kustoTables.Read())
+                    {
+                        tablesAndFunctions.Add(Convert.ToString(kustoTables[0]));
```

Comment:
Not sure what happens if there are no tables, exception?

Reply:
No exception if there are no rows to read, `Read()` will return false and no element will be added to the list.